airmozilla/main/templates/main/home.html

```diff
@@ -11,7 +11,17 @@
 </style>
 {% endblock %}
 
-
+{% block extra_head %}
+{{ super() }}
+<meta name="keywords" content="{{ ','.join(tags) }}">
+<meta property="og:title" content="Home">
```

Comment:
This should be the same title as this page which isn't Home. It's "Air Mozilla | Mozilla, in Video" but you shouldn't have to write that twice. There's got to be a way to write this one and use in multiple places.